docs/pages/concepts/networking/components/network-manager.md

```diff
@@ -19,3 +19,59 @@ The network parameters between two clusters are exchanged using the`networkconfi
 2. `NCFG 1->2` is replicated to the remote cluster by the `CRDReplicator`, running in Cluster1 who knows how to interact with the API server of Cluster2. Same step done in Cluster2;
 3. In Cluster2 the custom resource `NCFG (1->2)` is processed and its status updated with the NAT information if any. The same is done with `NCFG (2->1)` in Cluster1.
 4. The CRDReplicator running in Cluster1 reflects this changes in the local resource `NCFG(1->2)`. Same step done in Cluster2; The TunnelEndpointCreator in Cluster1 combining the status of `NCFG(1->2)` and the spec of `NCFG (2->1)` it has all the necessary information to create the custom resource `TEP (1-2)` of type `tunnelendpoints.net.liqo.io`.
+
```

Comment:
1) There are **5** steps in the diagram, **4** in the text. Please arrange the text accordingly to the 5 steps.
2) Modify the picture so that Liqo components (e.g., CRD replicators) are **within** the proper cluster. All the components should be **inside** their cluster, not outside.